recipe/build.sh

```diff
@@ -3,54 +3,105 @@
 # unset the SUBDIR variable since it changes the behavior of make here
 unset SUBDIR
 
-USE_NONFREE=no
-
 declare -a _CONFIG_OPTS=()
 
-# We do not care what the defaults are as they could change. Be explicit
-# about every flag.
-if [[ ${USE_NONFREE} == yes ]]; then
-  _CONFIG_OPTS+=("--enable-nonfree")
-  _CONFIG_OPTS+=("--disable-gpl")
-  _CONFIG_OPTS+=("--disable-gnutls")
-  _CONFIG_OPTS+=("--enable-openssl")
-  # The Cisco GPL-compliant wrapper (you need to get your own binaries for this)
+if [[ ${target_platform} != win-64 ]]
+then
+  ##### For platforms LINUX-*, OSX-* #####
+
+  # we choose libopenh264 instead of x264 to make this LGPL
+  # these codecs are not supported for win-64 at the moment
   _CONFIG_OPTS+=("--enable-libopenh264")
-  # GPL-3.0
-  _CONFIG_OPTS+=("--enable-libx264")
+  _CONFIG_OPTS+=("--enable-libopus")
+  _CONFIG_OPTS+=("--enable-libopenjpeg")
+  _CONFIG_OPTS+=("--enable-libvorbis")
+  _CONFIG_OPTS+=("--enable-libmp3lame")
+  _CONFIG_OPTS+=("--enable-pthreads")
+
+  ##### For platforms LINUX-64, LINUX-AARCH64
+  if [[ ${target_platform} == linux-64 ]] || [[ ${target_platform} == linux-aarch64 ]]
+  then
+     # libtesseract not supported on s390x and win-64
```

Comment:
Is it not supported by ffmpeg or by libtesseract or just that we don't have these packages on main? Right now the comment leaves a lot of space for interpretation.

Reply:
updated comments in this file